Userland/Libraries/LibWasm/AbstractMachine/Interpreter.cpp

```diff
@@ -230,14 +263,29 @@ struct ConvertToRaw<double> {
     }
 };
 
+Vector<NonnullOwnPtr<Value>> Interpreter::pop_values(Configuration& configuration, size_t count)
+{
+    Vector<NonnullOwnPtr<Value>> results;
```

Comment:
`NonnullOwnPtrVector<Value>`?

Reply:
iterating over that will give out references to Value instead of `NonnullOwnPtr<Value>`'s, so no.